### PR TITLE
Sort team selections

### LINKED ISSUE #4737: Sort team selections
### New Issue Checklist

- [x] Updated fastlane to the latest version
- [x] I have read the [Contribution Guidelines](https://github.com/fastlane/fastlane/blob/master/CONTRIBUTING.md)

### Issue Description

When presenting the list of teams for interactive selection (both for Developer Portal and iTunes Connect), the teams are listed in whatever order Apple sends them. When scanning long lists, it would be easier to find the correct team if they were alphabetized by name.

### Environment

fastlane version (run `fastlane -v`): fastlane 1.89.0

Do you use bundler to execute fastlane (i.e. `bundle exec fastlane`)? yes

Do you use a Ruby environment manager (e.g. `chruby`, `rbenv`, `rvm`)? rbenv

### PR DESCRIPTION
**Pull request by ravinggenius**
Fixes #4737.

### CODE REVIEW

File: spaceship/spec/UI/select_team_spec.rb [hunk 1]
```diff
@@ -23,13 +23,13 @@
 
         it "Lets the user select the team if in multiple teams" do
           allow($stdin).to receive(:gets).and_return("2")
-          expect(subject.select_team).to eq("SecondTeam") # a different team
+          expect(subject.select_team).to eq("XXXXXXXXXX") # a different team
```
Comment (KrauseFx) [inline spaceship/spec/UI/select_team_spec.rb]: This kind of makes the test less useful, could you add a second test that selects `"1"` and therefore selects `SecondTeam`

**Comment by KrauseFx** [comment]
Looks great, thanks for the PR. I added one comment before it's ready to be merged

File: spaceship/spec/UI/select_team_spec.rb (hunk 1 above)
Comment (ravinggenius) [inline spaceship/spec/UI/select_team_spec.rb]: Added second test, rebased and force-pushed.

**Comment by KrauseFx** [comment]
This is great, thanks @ravinggenius for your contribution 👍

**Pull request merged**